dist/templates/index.tmpl

```diff
@@ -0,0 +1,280 @@
+              </ul>
+
+              <form class="navbar-form visible-xs visible-sm" role="search">
+                  <div class="form-group">
+                      <select ng-model="nav.selected" ng-options="module as module.name for module in modules" class="form-control" ng-change="goToSelected(nav.selected)">
+                          <option value="" ng-init="defaultText = 'Go to module'" ng-bind="nav.selected.name || defaultText"></option>
+                      </select>
+                  </div>
+              </form>
+              <%= navContent %>
+          </div> <!-- collapse END -->
+        
+
+        </div> <!-- container END -->
+
+
+      </div> <!-- navbar END -->
+    </header>
+
+    <div role="main" class="container">
+      <div class="row clear-navbar"></div>
+
+      <div class="row">
+        <div class="col-xs-12">
+        <!--[if lt IE 7]>
+        <p class="alert alert-error">Your browser is <em>ancient!</em>
+          <a href="http://browsehappy.com/">Upgrade to a different browser</a> or
+          <a href="http://www.google.com/chromeframe/?redirect=true">install Google Chrome Frame</a> to
+          experience this site.
+        </p>
+        <![endif]-->
+
+        <!--[if lt IE 9]>
+        <div class="alert">
+          You are using an old version of Internet Explorer.
+          For better and safer browsing experience please <a href="http://www.microsoft.com/IE9">upgrade IE</a>
+          or install <a href="http://google.com/chrome">Google Chrome browser</a>.
+        </div>
+        <![endif]-->
+      </div>
+    </div>
+
+    <div class="row">
+
+      <!-- Sticky sidebar -->
+      <div class="col-xs-12 col-md-3 sidebar">
+          
+        <div class="form-group">
+            <input 
+                class="form-control" 
+                typeahead="module as module.name for module in modules | filter:$viewValue | limitTo:12" 
+                typeahead-on-select="goToSelected($item)"
+                type="text" ng-model="nav.selected" placeholder="search the docs" tabindex="1" accesskey="s" focused="focused">
+        </div>
+   
+        <ul class="minimal list-group hidden-xs hidden-sm" ng-repeat="module in modules track by module.url">
+            <a href="{{module.url}}"> {{module.name}} </a>
+            
+            <!-- Opening accordion -->
+            <li accordion class="list-group-item" close-others="true">
+
+                <!-- Directives -->
+                    <accordion-group ng-if="module.directives.length" is-open="status.isFirstOpen" is-disabled="status.isFirstDisabled">
+                        <accordion-heading>
+
+                            <span class="caret caret-west" ng-class="$parent.isOpen ? 'caret-south' : 'caret-west'"></span> <span class="small">Directives </span>
+                        </accordion-heading>
+                        <dl class="small" ng-repeat="page in module.directives track by page.url" ng-class="navClass(page)">
```

Comment:
hmmm this seems to be breaking for me. I am getting the class "duplicate keys" error. When I change this to `track by $index` it works ok.